### ISSUE TITLE
I wish the user could edit that ratio on the primary node.

### ISSUE THREAD

**Issue by Neytiri7** [Tropfchen/ComfyUI-yaResolutionSelector#3]
I'm using the "nodes" Python file with the desired percentage in editing.
If I just write down the ratio, it's calculated on its own, so I'm using it very comfortably...

As I proceeded with the update, the item continued to be initialized, and I had to edit the file continuously after the update.

So, even if you create a file that creates that ratio separately, set it at the ratio that the user wants, and update it, it would be good if the file did not change.

If you use the advanced node, you have to enter the percentage every time you enter your job, so you use the default node,

Among the ratios I usually use, there is a ratio of 4:5 years old...
It was not in this item, so I was editing and using the file.

If improvement is possible, please improve it.
Thank you.

**Comment by Tropfchen** [comment]
You change file "nodes.py", and it changes back every time you run update in ComfyUI-Manager?

**Comment by Neytiri7** [comment]
Yes, when I pressed Update, it was reset. And, I still try to update it to keep the latest version, but it keeps resetting.

So, it's uncomfortable right now, but I'm using it as an advanced node.

**Comment by Tropfchen** [comment]
Good to know, thanks. Added quick and dirty fix for that, on first run file ratios.txt is created in _ComfyUI\custom_nodes\ComfyUI-YetAnotherResolutionSelector_ , modify it instead. 
For now restarting comfy is required after modifying the file, don't forget to also force refresh of page with comfy (ctrl F5).
